Makefile

```diff
@@ -6,10 +6,10 @@ SHELL=/bin/bash -o pipefail -o errexit
 
 -include .env
 
-AIO_VERSION       ?= 4.10.6
+AIO_VERSION       ?= 4.10.7-rc1
 CONTAINER_VERSION ?= test
-ALPINE_VERSION    = 3.18
-NGINX_VERSION     = 1.24.0
+ALPINE_VERSION    = 3.20
+NGINX_VERSION     = 1.26.1
```

Comment:
For now, we are not ready to use `?=` unfortunately due to inconsistent variable names